src/screen.c

```diff
@@ -43,7 +43,12 @@ void bt_handler(bool connected) {
     } else {
         bool did_vibrate = persist_exists(KEY_BLUETOOTHDISCONNECT) ? persist_read_int(KEY_BLUETOOTHDISCONNECT): 0;
         if (is_bluetooth_vibrate_enabled() && !is_user_sleeping() && !did_vibrate) {
-            vibes_long_pulse();
+            static uint32_t const segments[] = { 200, 100, 100, 100, 500 };
```

Comment:
This is a good idea, but I think this pattern is pretty long and other users might not like it. It's a total of a whole second vibration pattern, so it might be a little too long. Maybe just {200, 100, 100} would be better?